azure-pipelines-official.yml

```diff
@@ -98,19 +106,21 @@ stages:
         arguments: '-sourceBranchName $(SourceBranchName) -prNumber $(PRNumber)'
       condition: and(succeeded(), ne(variables['PRNumber'], 'default'))
 
+    # Skip master validation in dnceng until the repo PAT issue is resolved
     - task: tagBuildOrRelease@0
       displayName: Tag master validation build
       inputs:
         type: 'Build'
         tags: |
           MasterValidationBuild
-      condition: and(succeeded(), eq(variables['SourceBranchName'], 'master'))
+      condition: and(succeeded(), eq(variables['SourceBranchName'], 'master'), eq(variables['System.TeamProject'], 'DevDiv'))
 
+    # Skip master validation in dnceng until the repo PAT issue is resolved
```

Comment:
@JoeRobich Skipping master validation in dnceng for now, since "git pull" fails to authenticate at the moment

Reply:
Didn't we just make a change that removed some github credentials from a script? Is there any connection between those changes?

For some reason I can't find the PR but I swear I just signed off on it and saw it get merged in the past few days @tmat.

Reply:
No, this is accessing roslyn repo in dnceng, not the github one. Anyways, I added another commit to address it